test/unit/lodash.spec.js

```diff
@@ -0,0 +1,275 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var chance = require('chance');
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash Collection training', function () {
+    describe('countBy', function () {
+        it('should count repeating numbers', function () {
+            expect(_.countBy.apply(_, datasets.countBy())).to.eql({1: 1, 2: 2, 6: 2, 7: 3});
+        });
+        it('should count length of given strings', function () {
+            expect(_.countBy.apply(_, datasets.countBy2())).to.eql({3: 2, 4: 3, 5: 1});
+        });
+    });
+
+    describe('every', function () {
```

Comment:
every: We are lacking _.matches iteratee, function predicate and once again we should make tests for use with different datatypes